cgt_calc/main.py

```diff
@@ -57,6 +58,55 @@ def get_amount_or_fail(transaction: BrokerTransaction) -> Decimal:
     return amount
 
 
+# Amount difference can be caused by rounding errors in the price.
+# Schwab rounds down the price to 4 decimal places
+#  so that the error in amount can be more than $0.01.
+# Fox example:
+# 500 shares of FOO sold at $100.00016 with $1.23 fees.
+# "01/01/2024,"Sell","FOO","FOO","500","$100.0001","$1.23","$49,998.85"
+# calculated_amount = 500 * 100.0001 - 1.23 = 49998.82
+# amount_on_record = 49998.85 vs calculated_amount = 49998.82
+def _approx_equal_price_rounding(
+    amount_on_record: Decimal,
+    quantity_on_record: Decimal,
+    price_on_record: Decimal,
+    fees_on_record: Decimal,
+    calcuationType: CalcuationType,
+) -> bool:
+    calculated_amount = Decimal(0)
+    calculated_price = Decimal(0)
+    if calcuationType is CalcuationType.ACQUISITION:
+        calculated_amount = Decimal(-1) * (
+            quantity_on_record * price_on_record + fees_on_record
+        )
+        calculated_price = (
+            Decimal(-1) * amount_on_record - fees_on_record
+        ) / quantity_on_record
+    elif calcuationType is CalcuationType.DISPOSAL:
+        calculated_amount = quantity_on_record * price_on_record - fees_on_record
+        calculated_price = (amount_on_record + fees_on_record) / quantity_on_record
+    if abs(calculated_price - price_on_record) < Decimal("0.0001"):
+        LOGGER.debug(
+            "Price calculated_price %.6f vs price_on_record %s in an acceptable range",
+            calculated_price,
+            price_on_record,
+        )
+        return True
+    LOGGER.debug(
+        "Price error in calculated_price %.6f vs price_on_record %s",
+        calculated_price,
+        price_on_record,
+    )
```

Comment:
This logs the same information as the lines below, can you please remove it?

Reply:
Updated